MerlinAU.sh

```diff
@@ -1447,8 +1447,12 @@ _CreateEMailContent_()
    then subjectStr="F/W Update Status for $node_lan_hostname"
    else subjectStr="F/W Update Status for $MODEL_ID"
    fi
-   fwInstalledVersion="$(_GetCurrentFWInstalledLongVersion_)"
-   fwNewUpdateVersion="$(_GetLatestFWUpdateVersionFromRouter_ 1)"
+   if ! "$offlineUpdateTrigger"
```

Comment:
This is the bug fix, just to make it easier on you :)

Reply:
> This is the bug fix, just to make it easier on you :)
>
Good catch!!